dojo/db_migrations/0191_finding_template_findings_to_replace_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1.10 on 2023-07-26 14:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0190_system_settings_experimental_fp_history'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='finding_template',
+            name='findings_to_replace',
+            field=models.TextField(blank=True, help_text='Title of findings you want to replace with this template separate by semicolon.', max_length=1000, null=True),
+        ),
+        migrations.AddField(
+            model_name='finding_template',
+            name='list_replace',
+            field=models.BooleanField(default=False, help_text='Apply template to all findings listed in "Findings to replace". (Update will overwrite all filled fields. You can merge field using {original} placeholder.)'),
+        ),
```

Comment:
I suppose these lines could be replaced by lines from `0192` and the whole `0192` can be removed.

Reply:
I agree, this could be merged.

Reply:
Hi ! 
Not sure to understand what you are talking about 😅.
If I can help on anything just tell me :)

Reply:
Like this https://github.com/Siryu6/django-DefectDojo/pull/1